src/Translatable/Translatable.php

```diff
@@ -260,6 +261,40 @@ public function fill(array $attributes)
     }
 
     /**
+     * Get the fillable attributes for the model.
+     *
+     * @return array
+     */
+    public function getFillable()
+    {
+        $fillable = parent::getFillable();
+        $locales = $this->getLocales();
+
+        foreach ($this->translatedAttributes as $attribute) {
+            if ($this->alwaysFillable() || $this->isFillable($attribute)) {
+                foreach ($locales as $locale) {
+                    $fillable[] = $this->getFillableLocalizedAttributeKey($attribute, $locale);
+                }
+                unset($fillable[array_search($attribute, $fillable)]);
+            }
+        }
+
+        return $fillable;
+    }
+
+    /**
+     * Get the localized attribute key for use in the $fillable array.
+     *
+     * @param string $key
+     * @param string $locale
+     * @return array
+     */
+    protected function getFillableLocalizedAttributeKey($key, $locale)
```

Comment:
Please do the following changes:

- make this method private. We want to reduce visibility as much as possible.
- change the order of the arguments. Let $locale be first to match the order of the returned string.
- rename $key to $attribute. Makes it more readable I think.

Would you agree renaming this to `getLocalizedAttributeAsFillableKey()`?